clients/runtime/src/error.rs

```diff
@@ -104,9 +104,8 @@ impl Error {
 	pub fn is_module_err(&self, pallet_name: &str, error_name: &str) -> bool {
 		matches!(
 			self,
-			Error::SubxtRuntimeError(SubxtError::Runtime(DispatchError::Module(ModuleError{
-				pallet, error, ..
-			}))) if pallet == pallet_name && error == error_name,
+			Error::SubxtRuntimeError(SubxtError::Runtime(DispatchError::Module(module_error
+			))) if module_error.details().unwrap().pallet.name() == pallet_name && &module_error.details().unwrap().variant.name == error_name,
```

Comment:
Let's refactor this statement so that we don't have to unrwap() twice and it becomes easier to read.

Reply:
We can use an inner match maybe.